### ISSUE TITLE
Chat / Question : AppleTV / tvOS Support for TwitchPlayer

### ISSUE THREAD

**Issue by mkral** [justintv/Twitch-API#446]
I've seen many apps extracting the m3u8 playlists from Twitch, but it's against the ToS. Unfortunately the new AppleTV doesn't allow WebViews which makes the embedded player unusable on tvOS without violating the ToS. 

I understand **WHY** twitch wants developers to use their player and show the ads but right now it's not possible. Are there any plans to add either a player that will work for tvOS or have some playlists that already include ads that wouldn't go against the ToS? 

Does Twitch grant any exceptions to the section 13.F?

**Comment by mkral** [comment]
Open sourcing the Twitch Player used in the iOS would also be awesome! Unless it's using web views.

**Comment by gdavis** [comment]
I read the other thread where you guys were talking about using the usher m3u8 URLs, have you made any headway in using the "official" TwitchPlayer for tvOS? I'm tinkering around with tvOS as well and curious if the TOS is blocking you guys as well.

**Comment by mkral** [comment]
@gdavis yes, there's currently no way to play twitch streams / videos without breaking ToS or using Apple's private APIs. It sucks, but hopefully someone caves first.

**Comment by gdavis** [comment]
Bummer. Hopefully they open the door for those of us that want to make cool tvOS apps...
